x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/console_commands_definition.ts

```diff
@@ -421,5 +442,68 @@ export const getEndpointConsoleCommands = ({
     });
   }
 
+  // `execute` is currently behind feature flag
+  // planned for 8.8
+  if (isExecuteEnabled) {
+    consoleCommands.push({
+      name: 'execute',
+      about: getCommandAboutInfo({
+        aboutInfo: i18n.translate('xpack.securitySolution.endpointConsoleCommands.execute.about', {
+          defaultMessage: 'Execute a command on the host',
+        }),
+        isSupported: doesEndpointSupportCommand('execute'),
+      }),
+      RenderComponent: ExecuteActionResult,
+      meta: {
+        endpointId: endpointAgentId,
+        capabilities: endpointCapabilities,
+        privileges: endpointPrivileges,
+      },
+      exampleUsage: 'execute --command "ls -al" --timeout 2s --comment "Get list of all files"',
+      exampleInstruction: ENTER_OR_ADD_COMMENT_ARG_INSTRUCTION,
+      validate: capabilitiesAndPrivilegesValidator,
+      mustHaveArgs: true,
+      args: {
+        command: {
+          required: true,
+          allowMultiples: false,
+          about: i18n.translate(
+            'xpack.securitySolution.endpointConsoleCommands.execute.args.command.about',
+            {
+              defaultMessage: 'The command to execute',
+            }
+          ),
+          validate: (argData) => {
+            return emptyArgumentValidator(argData);
+          },
+        },
```

Comment:
`validate` is no longer needed. There is [a new property on this command definition named `mustHaveValue`](https://github.com/elastic/kibana/blob/9ac065ab02ba729f5b44117c9f95e8ddd55866fe/x-pack/plugins/security_solution/public/management/components/console/types.ts#L30-L42). Set that to `non-empty-string` instead.

Reply:
🤯 nice

Reply:
997bde19c8da4e7798793f8fcd7041902412a6f1

Reply:
I'm not seeing the `mustHaveValue` setting here on `command` argument. Looks like you added it only to `timeout`

Reply:
fdac60dd7e5fc6cb3b807d4b10534da927e12fb3